src/main/java/com/softserveinc/ita/utils/Properties.java

```diff
@@ -0,0 +1,7 @@
+package com.softserveinc.ita.utils;
+
+public class Properties {
```

Comment:
probably better to use configuration.properties and create util class for working with it